src/translations/en.json

```diff
@@ -446,5 +457,27 @@
     "404": {
       "title": "404 Not Found"
     }
+  },
+  "callToAction" :{
+    "betaTestTitle" :"Beta Test in Progress: Click to learn more",
+    "betaTestAddition" : "about the revised 5's rule change",
+    "revisedFives": "Revised 5's",
+    "revisedFivesAddition": "Rule change",
+    "testDates": "Test Dates: April",
+    "betaTest": "Beta Test",
+    "ruleInfoHeader":"Rule Information",
+    "ruleInfo": "Playing a 5 now has a revised one-off effect: discard a card, then draw three cards ",
+    "whyImplementHeader" : "Why are we implementing this?",
+    "whyImplement" : "Because 5's take your turn to play and you could alternatively draw a card on your turn, while playing a 5 feels like it gets you two cards(yay!) it effectively only replaces itself without doing anything else for you. In order to power up the five and give it a specific purpose, while leaning into the them of 5's cycling your hand to draw you lots of cards, we're experimenting with a catiation on the 5's effect where you first discard a card from your hand, then draw three cards! We hope this will increase the usefullness of the 5's effect and hone it's strategic function. And we need your help deciding if this should become a permanent change!",
```

Comment:
```suggestion
    "whyImplement" : "Because 5's take your turn to play and you could alternatively draw a card on your turn, while playing a 5 feels like it gets you two cards (yay!) it effectively only replaces itself without doing anything else for you. In order to power up the five and give it a specific purpose, while leaning into the theme of cycling your hand to draw you lots of cards, we're experimenting with a variation on the 5's effect where you first discard a card from your hand, then draw three cards! We hope this will increase the usefulness of the 5's effect and hone it's strategic function. And we need your help deciding if this should become a permanent change!",
```